packages/scripts/webapp/preset/config/webpack.config.js

```diff
@@ -37,19 +34,19 @@ function getCommonStyleLoaders(enableModules, mode) {
 	let cssOptions = {};
 	if (enableModules) {
 		cssOptions = {
-			sourceMap: mode !== 'production',
+			sourceMap: mode === 'development',
 			modules: true,
 			importLoaders: 1,
 			localIdentName: '[name]__[local]___[hash:base64:5]',
 		};
 	}
 	return [
-		{ loader: MiniCssExtractPlugin.loader },
+		{ loader: mode === 'development' ? 'style-loader' : MiniCssExtractPlugin.loader },
 		{ loader: 'css-loader', options: cssOptions },
 		{
 			loader: 'postcss-loader',
 			options: {
-				sourceMap: mode !== 'production',
+				sourceMap: mode === 'development',
```

Comment:
DRY, you can use a variable 😎

Reply:
It was more to not having mode/isProdMode/isDevMode depending on where you are.
Here the args are consistent.
:/